### PR TITLE
Aligned README with Discovery 2.0

### PR DESCRIPTION
**Pull request by lzap**

### CODE REVIEW

**Comment by shlomizadok** [comment]
I have installed my discovery plugin with those instructions. They are much better and installation went smoothly.

**Comment by orrabin** [comment]
Installer directions also work

File: README.md [hunk 1]
```diff
@@ -183,7 +135,7 @@ configuration is available.
 If Locations and/or Organisations are enabled, Foreman will default to using
 the first Location and first Organisation for Discovered hosts. If hosts should
 be placed in some other Location/Organization, alter the default Loc/Org in
-`More->Settings->Discovery Settings`
+`More->Settings->Discovery`
```
Comment (orrabin) [inline README.md]: should be `More->Settings->Discovered`

**Comment by lzap** [comment]
Thanks for testing, once you are fine with the process, please go ahead :-)

File: README.md (hunk 1 above)
Comment (lzap) [inline README.md]: Oh, right. Thanks.

**Comment by lzap** [comment]
Pushed....

**Comment by orrabin** [comment]
Thanks @lzap merged as 69c30f143144d5ac43c3f0db910c7d1045f0bc59

**Pull request closed without merging**